### PR TITLE
Docs fixes based on UX studies

### PR DESCRIPTION
**Pull request by luise**

### CODE REVIEW

File: cli/command/init/prompter.js [hunk 1]
```diff
@@ -282,6 +282,12 @@ function machineConfigPrompts(provider) {
   const sizeChoices = getInquirerDescriptions(provider.getSizes());
   sizeChoices.push({ name: consts.other, value: consts.other });
 
+  const sizeOptions = [{
+    name: `${consts.instanceTypeSize} (recommended)`,
+    value: consts.instanceTypeSize },
+  { name: consts.ramCpuSize },
```
Comment (HongjianZhang) [inline cli/command/init/prompter.js]: Not sure if the this line needs to be indented.

Comment (luise) [reply cli/command/init/prompter.js]: I thought so too! But the linter corrects it to this indentation level... I agree it looks weird though.

Comment (kklin) [reply cli/command/init/prompter.js]: Maybe it'll be prettier if you put the `}` on line 287 on a new line?

Comment (kklin) [reply cli/command/init/prompter.js]: The `{` on line 285 may need to go on a new line as well. dunno

Comment (HongjianZhang) [reply cli/command/init/prompter.js]: Yeah. I think that would be a good idea.

Comment (luise) [reply cli/command/init/prompter.js]: Agreed! Found a way that looks good and makes the linter happy. I'll update

Comment (luise) [reply cli/command/init/prompter.js]: @HongjianZhang , updated! Thanks!

File: cli/command/init/prompter.js [hunk 2]
```diff
@@ -356,6 +364,7 @@ function machineConfigPrompts(provider) {
         return answers[consts.sizeType] !== consts.instanceTypeSize;
       },
     },
+    'For small applications, 2 GiB is a suitable choice.'),
```
Comment (kklin) [inline cli/command/init/prompter.js]: Super nit - above you have `}, 'help'`, while here you have `},
'help'`. It'd be nice to be consistent, I don't really care which one you go with

**Comment by kklin** [comment]
This needs to be rebased now

**Comment by luise** [comment]
@kklin , rebased!

**Pull request merged**